data/card_catalogue/unique-1.json

```diff
@@ -264,27 +184,17 @@
     "upgrade": "",
     "tier": "green",
     "details": {
-      "rules": "Unarmed attacks deal double damage until you play your next card.",
+      "rules": "Attacks deal double damage until you play your next card.",
       "flavour": "I'm not unarmed, I've got two of them.",
       "image": "assets/core/card/illustration/Bashbolster.png"
     }
   },
-  {
-    "title": "Glass Hammer",
-    "upgrade": "",
-    "tier": "blue",
-    "details": {
-      "rules": "Deal 10 damage to a visible target. Return this card to the deck after casting it.",
-      "flavour": "Bonk",
-      "image": "assets/core/card/illustration/Glass Hammer.png"
-    }
-  },
   {
     "title": "Musty Musket",
     "upgrade": "",
     "tier": "blue",
     "details": {
-      "rules": "Flip a coin. Heads: Deal 10 damage to visible target. Tails: Gun backfires, take 10 damage.",
+      "rules": "Flip a coin. Heads: Deal 15 damage to visible target. Tails: Gun backfires, take 5 damage.",
```

Comment:
This was just too punishing IMO. I wouldn't want to deal with a character getting one pieced by this failing.

Reply:
5 damage is already pretty high stakes